login-workflow/src/components/RegistrationWorkflow/RegistrationWorkflow.tsx

```diff
@@ -23,9 +23,25 @@ import { parseQueryString } from '../../utils';
  */
 
 export type RegistrationWorkflowProps = {
+    /**
+     * The initial screen index to start the registration workflow from
+     */
     initialScreenIndex?: number;
+
+    /**
+     * The success screen to display upon successful registration
+     */
     successScreen?: JSX.Element;
+
+    /**
+     * Boolean when true verifies validateUserRegistrationRequest for verifyCode
```

Comment:
This description may need a little cleaning up. Should mention that invite registration truncates the default list of screens to skip email entry and validation and pulls the validation code from the url query string instead.